src/Symfony/Bundle/FrameworkBundle/Translation/Translator.php

```diff
@@ -46,6 +46,8 @@ class Translator extends BaseTranslator implements WarmableInterface
     private $resources = array();
 
     /**
+     * Constructing new translator.
```

Comment:
Those changes are indeed adding dummy content, but let us look up why.
First line or two of phpdoc are Summary, later we do have Description. Summary shall end with full stop and be followed by empty line, or it could not be ended with full stop, yet there it shall be 2 empty lines.

```
Available options:

* foo
* bar
```
are not fulfilling that condition.

So what is this content? This content is Description, not a Summary, yet if one want to add Description, Summary section is obligatory.
Violating this rule is problematic for automated extraction of Description and Summary. So let us fulfill the standard, having PHP CS Fixer violations fixed as a bonus.

Reply:
I was never fan of this rule and you can see now why. I don't see how does enforcing dots at the end of summary improve anything. Am I the only one?

Reply:
Usually Symfony uses simple `Constructor.` 👍

Reply:
Indeed, we are using `Constructor.` everywhere.

Reply:
@ostrolucky , it's not about enforcing dots, it's about violating definition of Summary vs Description of phpdoc standard.

will move to `Constructor` indeed.